Fails on darwin.

```
build flags: SHELL=/nix/store/lah3ca47lhzpqfj33wwn48sp3x7g4l43-bash-4.4-p23/bin/bash
clang++ -DPACKAGE_NAME=\"Sta\" -DPACKAGE_TARNAME=\"sta\" -DPACKAGE_VERSION=\"0.1\" -DPACKAGE_STRING=\"Sta\ 0.1\" -DPACKAGE_BUGREPORT=\"simon.c.carter@gmail.com\" -DPACKAGE_URL=\"https://github.com/simonccarter/sta\" -I.     -g -O2 -c -o src/main.o src/main.cpp
src/main.cpp:90:49: error: cannot take the address of an rvalue of type 'std::__1::basic_ostringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
                        string String = static_cast<ostringstream*>( &(ostringstream() << ii->first) )->str();```

--- a/pkgs/tools/misc/sta/default.nix
+++ b/pkgs/tools/misc/sta/default.nix
@@ -1,0 +1,30 @@
+{ stdenv, lib, fetchFromGitHub, autoreconfHook }:
+stdenv.mkDerivation {
+  pname = "sta";
+  version = "unstable-2016-01-25";
+
+  src = fetchFromGitHub {
+    owner = "simonccarter";
+    repo = "sta";
+    rev = "2aa2a6035dde88b24978b875e4c45e0e296f77ed";
+    sha256 = "05804f106nb89yvdd0csvpd5skwvnr9x4qr3maqzaw0qr055mrsk";
+  };
+
+  buildInputs = [ autoreconfHook ];
+
+  meta = with lib; {
+    description = "Simple statistics from the command line interface (CLI), fast";
+    longDescription = ''
+      This is a lightweight, fast tool for calculating basic descriptive
+      statistics from the command line. Inspired by
+      https://github.com/nferraz/st, this project differs in that it is written
+      in C++, allowing for faster computation of statistics given larger
+      non-trivial data sets.
+    '';
+    license = licenses.mit;
+    homepage = "https://github.com/simonccarter/sta";
+    maintainers = with maintainers; [ infinisil ];
+    platforms = platforms.all;
+    badPlatforms = platforms.darwin;
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -5103,6 +5103,8 @@ in
 
   multitime = callPackage ../tools/misc/multitime { };
 
+  sta = callPackage ../tools/misc/sta {};
+
   multitran = recurseIntoAttrs (let callPackage = newScope pkgs.multitran; in {
     multitrandata = callPackage ../tools/text/multitran/data { };
 

